storage/test/tuples.go

```diff
@@ -457,3 +458,59 @@ func TuplePaginationOptionsTest(t *testing.T, datastore storage.OpenFGADatastore
 		}
 	})
 }
+
+func ReverseReadTuplesTest(t *testing.T, datastore storage.OpenFGADatastore) {
+
+	require := require.New(t)
+	ctx := context.Background()
+
+	storeID, err := id.NewString()
+	require.NoError(err)
+
+	err = datastore.Write(
+		ctx,
+		storeID,
+		nil,
+		[]*openfgapb.TupleKey{
+			tuple.NewTupleKey("document:doc1", "viewer", "user:jon"),
+			tuple.NewTupleKey("document:doc2", "viewer", "group:eng#member"),
+			tuple.NewTupleKey("document:doc3", "editor", "user:jon"),
```

Comment:
can we add another tuple here with a type that is not `document`?

Reply:
Yeah, good point.

Reply:
Updated.